docs/_scripts/sync.sh

```diff
@@ -3,7 +3,7 @@
 # Sync repository foo using remote bar
 echo "Create a task to sync the repository using the remote."
 export TASK_URL=$(http POST $BASE_ADDR$REPO_HREF'sync/' \
-    remote=$REMOTE_HREF \
+    remote=$REMOTE_HREF skip:="[\"src\"]" \
```

Comment:
Would there be any reason for us to hold off utilizing the ':=' assignment operator?  It's brand new to Python as of 3.8.  Are we sure none of our users would be on older versions of Python than the very latest?

Reply:
This isn't python related, it's an ```httpie``` thing.  ```:=``` is used for providing raw JSON.  https://httpie.org/doc#request-items

Reply:
D'oh!  You're right @dralley, thanks for the catch

Reply:
Yes as in this PR it will use List field we have to use this one (or we got type error, string instead of list)